packages/astro/src/core/app/index.ts

```diff
@@ -184,35 +184,8 @@ export class App {
 			}
 			Reflect.set(response, responseSentSymbol, true);
 			return response;
-		} else {
```

Comment:
wow, love all of this deletion

Reply:
Agreed, all this cleanup in the `App` is great!